spec/addressable/idna_spec.rb

```diff
@@ -249,11 +255,6 @@
       "example..host"
     )).to eq("example..host")
   end
-
-  it "should normalize 'string' correctly" do
-    expect(Addressable::IDNA.unicode_normalize_kc(:'string')).to eq("string")
-    expect(Addressable::IDNA.unicode_normalize_kc("string")).to eq("string")
-  end
```

Comment:
This function is no longer needed and there's no need to test ruby own implementation.